Makefile

```diff
@@ -61,8 +61,9 @@ test: manifests generate fmt vet ## Run unit tests
 ##@ Build
 
 GO=GO111MODULE=on GOFLAGS=-mod=vendor CGO_ENABLED=0 go
-build: generate fmt vet ## Build manager binary.
+buildbin:
```

Comment:
can we call this `build-manager` instead? 

And to be clear this new directive is just to avoid copying the `$(GO)` call into `Dockerfile.openshift`?
If so, can you put a comment above `buildbin` explaining it's purpose?

Reply:
Sure, will call it `build-manager`

Yes, to avoid the copying, and I didn't want to use `build` target as it depends on `generate` target which generates some code.